modules/internal/Bytes.chpl

```diff
@@ -1128,6 +1128,28 @@ module Bytes {
     doAppend(lhs, rhs);
   }
 
+  /*
+     Appends the bytes stored in the `rhs` tuple to the :type:`bytes` `this`.
+   */
+  proc ref bytes.appendBytes(bytesTup) : void
```

Comment:
When/if we're having design discussion for this, I will vote against this name :)

FWIW, the best alternative I can think of at the moment is an `appendByte(x: uint(8) ...)`

Reply:
Maybe we can just have the varargs version and call it simply `append` ?

Reply:
I'd find it confusing. Rhetorically, why not just add an overload of `+=` instead of adding a new `append`?

`"myString" += 65` could easily be conceived as changing the string to `"myString65"`. I admit that `append` is a bit clearer, but not sufficiently enough to me.

Reply:
I picked `append` simply because that's what Python names this operation.

I really don't like the idea of `myString += 65`. Can you suggest one or two alternative names that you do like? Does `appendByteValues` / `appendCodepoints` work for you?

Reply:
My biggest source of distaste is "bytes" in the name can be confused with `bytes` the type.

Another personal preference is that I am not too crazy about having both a singular and a plural version of these. I am OK with a singular-named method taking multiple things or vice versa.

Given all that, my order of preferences:

1. Just `appendByte` and `appendCodepoint` (varargs, no plural versions)
2. Just `appendByteValues` and `appendCodepointValues` (varargs, no singulars, consistent naming, decimal representation of codepoints are hard to come by, so "values" make it clear(?))
2a. Use `Vals` instead of `Values` to make it shorter
3. Just `appendBytes` and `appendCodepoints` (varargs, no singular versions)

> Does appendByteValues / appendCodepoints work for you?

I could make this the 4th option as I don't like the inconsistency.